boards/arm/wio_lte_cat_1/wio_lte_cat_1.yaml

```diff
@@ -0,0 +1,16 @@
+identifier: wio_lte_cat_1
+name: WIO-LTE-CAT.1
+type: mcu
+arch: arm
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+ram: 192
+flash: 1024
+supported:
+  - gpio
+  - i2c
+  - counter
+  - spi
+  - usb_device
```

Comment:
spi and usb_device are not enabled in dts.

Reply:
Will remove. USB is patches on top, but it's flaky so not included yet.